.travis.yml

```diff
@@ -0,0 +1,23 @@
+dist: trusty
+sudo: false
+language: node_js
+notifications:
+  email: false
+node_js:
+  - "8"
```

Comment:
Suggest to add `"10"` too, and run the release stage with this version (10 is LTS as of October 2018)